src/react-vite/02-setting-up-your-environment/setting-up-your-environment.md

```diff
@@ -152,55 +116,41 @@ npm install
 
-The development server is a useful tool. When it starts, it
-[transpiles](https://en.wikipedia.org/wiki/Source-to-source_compiler) the TypeScript and JSX code
-into JavaScript and bundles it for delivery to the browser. It also watches for changes to the
-source code, and when they occur, it repeats the process, then causes the browser to reload with
-those changes.
+The development server is a useful tool. When it starts, it [transpiles](https://en.wikipedia.org/wiki/Source-to-source_compiler) the TypeScript and JSX code into JavaScript and bundles it for delivery to the browser. It also watches for changes to the source code, and when they occur, it repeats the process, then causes the browser to reload with those changes.
 
 ✏️ Run the command:
 
 ```shell
 npm run dev
 ```
 
-The server will output the "Local" URL for the dev server, copy and paste it
-into a browser. The browser will display a welcome page with "Vite + React."
+The server will output the "Local" URL for the dev server, copy and paste it into a browser. The browser will display a welcome page with "Vite + React."
 
 <strong>Having issues with your local setup?</strong> See the solution in [StackBlitz](https://stackblitz.com/fork/github/bitovi/academy/tree/main/exercises/react-vite/02-setting-up-your-environment/02-solution?file=src/App.tsx) or [CodeSandbox](https://codesandbox.io/p/devbox/github/bitovi/academy/tree/main/exercises/react-vite/02-setting-up-your-environment/02-solution?file=src/App.tsx).
 
 ## Objective 3: Add testing infrastructure
 
 ### Testing code with React Testing Library
 
-We will create unit tests for our application using the [React Testing
-Library](https://testing-library.com/docs/react-testing-library/intro/). The
-unit tests for components will be focused on verifying that a component creates
-the correct DOM output based on the state the component is provided. We also
-need to create tests for any React custom Hooks we create to ensure they provide
-the expected results.
+We will create unit tests for our application using the [React Testing Library](https://testing-library.com/docs/react-testing-library/intro/). The unit tests for components will be focused on verifying that a component creates the correct DOM output based on the state the component is provided. We also need to create tests for any React custom Hooks we create to ensure they provide the expected results.
 
 ### Setup 3
 
-Before we can create or execute any tests, run the following command to
-install the packages related to the React Testing Library:
+Before we can create or execute any tests, run the following command to install the packages related to the React Testing Library:
 
 ✏️ Run:
 
 ```shell
-npm install --save-dev @testing-library/jest-dom@6 @testing-library/react@14 @testing-library/user-event@14 jsdom@24 vitest@1
+npm install --save-dev vitest@1 @testing-library/react@14 @testing-library/user-event@14 @testing-library/jest-dom@6 jsdom@24
```

Comment:
Why was this changed?

Reply:
To make it easier to explain in the training.